azure-pipelines.yml

```diff
@@ -99,10 +99,10 @@ stages:
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
             name: NetCorePublic-Pool
-            queue: BuildPool.Server.Amd64.VS2019.Open
+            queue: BuildPool.Server.Amd64.VS2019.Pre.Open
```

Comment:
@dougbu FYI

Reply:
This is exactly what I was worried about when Arcade updated the minimum required VS version without ensuring the agents had that version. The `xcopy` version of `msbuild` doesn't include C++ requirements and doesn't work for us at all. Agree we're really busted unless we move to the preview queue but this should **not** have happend.

/cc @ilyas1974 @MattGal

Reply:
The other option, since Arcade is breaking us, is to revert the Arcade changes and disable the subscription (using `darc`) until the agents are updated in the regular queue. Note the regular queue has more resources.

@Tratcher is there a dotnet/aspnetcore Arcade update that also fails due to the forced VS 16.8 requirement❔ We build _only_ native projects and installers using desktop `msbuild` in that repo and have disabled all post-build steps, meaning it is fine using 16.whatever.

Any other thoughts @Pilchie❔

Reply:
AspNetCore's SDK was updated a while ago. https://github.com/dotnet/aspnetcore/pull/25815

Reply:
As for queue resources, don't they get regularly reallocated based on load?

Reply:
Queues are rebalanced approximately weekly last I checked.

aspnetcore wasn't affected by the SDK update because we make very limited use of desktop `msbuild`. Not sure why an Arcade update requiring 16.8 wouldn't break it though.